src/helpers/utils.tsx

```diff
@@ -426,31 +426,19 @@ export function getColor(taskObject: InitialTask): Color {
     }
   }
 }
-/** Transforms values of certain keys to the specified value
- * {T} obj - object where changes will be made
- * {string[]} propertyNames - list of property names in obj whose values will change
- * {any} newValue - the value to be assigned the property names
- * {any[]} old_value - replace property name if old_value is among specified here
+
+/** filters out plans whose jurisdiction id is null
+ * @param {Plan []} plansArray - a list of pre-filtered plans
+ *
+ * @return {Plan []} - A list of filtered plans
  */
-export function transformValues<T>(
```

Comment:
@p-netm could you add a little note in this PR for the reason why we removed this?

Reply:
ok